pkg/transform/txscript.go

```diff
@@ -99,12 +99,17 @@ type SourceAnnotations struct {
 // NodeTransformAnnotations contains a term, and one or more annotations
 type NodeTransformAnnotations map[string]interface{}
 
-func (nd *NodeTransformAnnotations) setProperties(mv map[string]*ls.PropertyValue) {
+func (nd *NodeTransformAnnotations) setProperties(mv map[string]interface{}) {
 	*nd = make(map[string]interface{})
 	for k, v := range mv {
 		u, err := url.Parse(k)
 		if err != nil || u.IsAbs() {
-			(*nd)[k] = v
+			switch v.(type) {
+			case string:
+				(*nd)[k] = ls.StringPropertyValue(k, v.(string))
+			case []interface{}:
+				(*nd)[k] = ls.StringSlicePropertyValue(k, v.([]string))
```

Comment:
This will panic. You have to create a []string, and copy one by one. Use something like this:
```
for _,x:=range v {
   slice=append(slice,fmt.Sprint(x))
}